lib/grplatt.gi

```diff
@@ -923,7 +923,8 @@ InstallGlobalFunction(LatticeViaRadical,function(arg)
       HN:=Image(hom,H);
       c:=LatticeByCyclicExtension(f,
 	  [u->IsSubset(HN,u),u->IsSubset(HN,u)])!.conjugacyClassesSubgroups;
-    elif select<>fail and (select=IsPerfectGroup  or select=IsSimpleGroup) then
+    elif select<>fail and (select=IsPerfectGroup  or select=IsAlmostSimpleGroup)
```

Comment:
This change goes beyond what the description of this PR. Is it intentional? If so, I of course trust that it is the right thing; I just wanted to verify, as it differed from the other changes.

Reply:
Thanks for noticing.  Yes, it needs to to be `NonabelianSimple`. Fixed.